chcemvediet/static/main/css/navs.sass

```diff
@@ -24,7 +21,9 @@
       &:focus,
         border: none
         color: $chv-color-blue
+  > li
```

Comment:
@martinmacko47 Tu som chcel pouzit rovnaky css selektor ako je [sem](https://github.com/martinmacko47/chcemvediet/blob/bug/380/fix-baseline-in-with_navigation/vendor/bootstrap_sass/static/stylesheets/_navs.scss#L130). Tento margin som potreboval prepisat.

Reply:
@viliambalaz Ok. Ale preco si potreboval novy blok s `> li` nestacilo sproste zanorit predosly `> li`?

```
.chv-nav
  border: ...
  ...
  > li
    position: relative
    ...
    + li
      ...
```

Reply:
dakujem, :ok: